server/middleware/checkPrisonerInCaseloadMiddleware.ts

```diff
@@ -23,7 +23,7 @@ export default function checkPrisonerInCaseload({
     const inactiveBooking = ['OUT', 'TRN'].some(prisonId => prisonId === prisonerData.prisonId)
 
     if (inactiveBooking) {
-      if (!(allowInactive && canViewInactiveBookings) && !(allowGlobal && globalSearchUser)) {
+      if (!(allowInactive && canViewInactiveBookings)) {
```

Comment:
I haven't included a test for this because there wasn't an existing test for the global search override behaviour to remove